.github/workflows/ci.yml

```diff
@@ -1,8 +1,6 @@
 on:
   pull_request:
     branches: [ master ]
-  schedule:
-    - cron: '00 18 * * *'
```

Comment:
We want to run the CI job daily to make sure we run against the latest `stable`, `beta` and `nightly` Rust and the latest `macos-latest` and `windows-latest`.

aside: we should probably use `ubuntu-latest` rather than `ubuntu-20.04`, worth fixing i'd say.